src/rez/package_test.py

```diff
@@ -394,6 +398,9 @@ def run_test(self, test_name):
             else:
                 command = map(variant.format, command)
 
+            if extra_test_args:
+                command = "{} {}".format(command, " ".join(extra_test_args))
```

Comment:
At this point in the code, `command` can be a string or a list of string. Additionally, arguments need to be properly quoted/escaped if `command` is a list of string. See https://github.com/AcademySoftwareFoundation/rez/pull/1523/files#diff-cb9e312c8b2786e57bdb95e7da7efe10407758af514dffb3ae1805f0fc996797R410-R413.

Reply:
Ah, I should've spotted that, thank you! Will take a look at this.

Reply:
Hey @JeanChristopheMorinPerso, sorry just trying to orient myself here; we would expect `command` to be a list of strings in the code here here if the `command` entry in the `tests` package definition is a list, E.G:
```
name = "test_package"
version = "1.0.0"
build_command = "cp -r {root}/tests {install_path}/tests"
tests={
    "unit": {
        "command": ["python3", "-m", "pytest", "{root}/tests"],  # <-- command is a list
        "requires": ["pytest"],
        "run_on": ["default"]
    }
}
```
have I got that right?

Reply:
I think so yes.

Reply:
I think I might have found a little bug with the behaviour of rez test when the command entry is a list but I think that it's probably out of scope here, I'll make an issue separately.